types/swagger-parser/index.d.ts

```diff
@@ -1,7 +1,54 @@
 // Type definitions for swagger-parser 4.x
 // Project: https://www.npmjs.com/package/swagger-parser
 // Definitions by: Tobias Wolff <https://github.com/Tobias4872/>
+//                 Eric Henry <https://github.com/EricHenry/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+import { Spec as Swagger } from '../swagger-schema-official';
```

Comment:
@sandersn thanks! glad I could contribute. Thanks for the guidance and helpful feedback! @minestarks